source/_components/lock.nuki.markdown

```diff
@@ -45,3 +45,9 @@ lock:
     port: 8080
     token: fe2345ef
 ```
+## {% linkable_title Services %}
```

Comment:
Suggest that this is rewritten in the same form as the [Apple TV component](https://home-assistant.io/components/media_player.apple_tv/#services).